src/popup/pages/SecureLoginSettings.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <IonPage>
+    <IonContent class="ion-padding ion-content-bg">
+      <div class="secure-login-settings">
+        <div class="top-wrapper">
+          <p class="text-description">
+            {{ $t('pages.secureLogin.description') }}
+          </p>
+
+          <SwitchButton
+            :label="$t('pages.secureLogin.enableSecureLogin')"
+            :model-value="isSecureLoginEnabled"
+            @update:modelValue="setSecureLoginEnabled"
```

Comment:
Maybe it would be possible to use:
```suggestion
            v-model="isSecureLoginEnabled"
```

Reply:
Since it is a computed value we cannot